lib/ancestry/class_methods.rb

```diff
@@ -125,25 +125,25 @@ def check_ancestry_integrity!(options = {})
           # ... check validity of ancestry column
           if !node.sane_ancestor_ids?
             raise Ancestry::AncestryIntegrityException, I18n.t("ancestry.invalid_ancestry_column",
-                                                               :node_id => node.id,
+                                                               :node_id => node.ancestry_identifier_column,
                                                                :ancestry_column => node.read_attribute(node.class.ancestry_column))
           end
           # ... check that all ancestors exist
           node.ancestor_ids.each do |ancestor_id|
             unless exists?(ancestor_id)
               raise Ancestry::AncestryIntegrityException, I18n.t("ancestry.reference_nonexistent_node",
-                                                                 :node_id => node.id,
+                                                                 :node_id => node.ancestry_identifier_column,
                                                                  :ancestor_id => ancestor_id)
             end
           end
           # ... check that all node parents are consistent with values observed earlier
-          node.path_ids.zip([nil] + node.path_ids).each do |node_id, parent_id|
-            parents[node_id] = parent_id unless parents.key?(node_id)
-            unless parents[node_id] == parent_id
+          node.path_ids.zip([nil] + node.path_ids).each do |node_identifier, parent_identifier|
+            parents[node_identifier] = parent_identifier unless parents.key?(node_identifier)
+            unless parents[node_identifier] == parent_identifier
```

Comment:
maybe we should leave as `node_id, parent_id` -- otherwise this is pretty verbose. What do you think? (I may change my mind later. honestly asking your opinion)

Reply:
Thank you for your comment. Here's how I'm currently thinking about this:

- I don't plan to change the naming of interface-level fields like parent_id or node_id.

- These names are compatible with existing behavior, and semantically they make sense as the identifiers used within the context of ancestry gem. 

For this reason, I’ve reverted instances of parent_identifier back to parent_id. I'd appreciate your thoughts on this direction.